.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.21-openshift-4.16
+  tag: rhel-9-release-golang-1.21-openshift-4.16
```

Comment:
If this is backported, this should probably be a 4.15 image ref?

Reply:
yeah potentially. It was 4.16 before so I didn't change it but don't mind doing the update

Reply:
There is no builder for `rhel9-golang-1.21-openshift-4.15`